lib/manager-webpack5/src/utils/manager-cache.ts

```diff
@@ -14,6 +14,12 @@ export const useManagerCache = async (
   options: Options,
   managerConfig: webpack.Configuration
 ) => {
+  const coreConfig = await options.presets.apply<CoreConfig>('core');
+  if ((coreConfig.builder as Webpack5BuilderConfig).options?.fsCache) {
+    logger.line(1); // force starting new line
+    logger.info('=> Ignoring Storybook cache because webpack cache is being used');
+  }
```

Comment:
Hmmm, I'm not sure about this. The manager cache is probably a bigger speedup than the WP cache... 🤔

Reply:
Yes, in my informal testing, i think the storybook manager cache was about 2 seconds faster than webpack. I thought we might still want to prefer webpack since it's more official, and less code for storybook to maintain. 
Let me know if you definitely want to remove it.

Reply:
I'm really not sure! Sorry if I was a bit terse, I wasn't necessarily disagreeing with this, just pondering. WDYT @ghengeveld / @shilman? 

I agree simplicity is better, I would definitely say we should only have one cache on the manager side, either WP's or our own.